### ISSUE TITLE
Add a link to blog article about usage

### ISSUE THREAD

**Issue by cowtowncoder** [indeedeng/lsmtree#8]
I may be wrong, but I think this entry:

https://engineering.indeedblog.com/blog/2013/10/serving-over-1-billion-documents-per-day-with-docstore-v2/

talks about system that uses lsmtree as a building block. If so, would be good to add a link from README (I can do a PR).

**Comment by youknowjack** [comment]
Good idea! A PR would be great. That blog post is more or less a written version of [this tech talk](https://engineering.indeedblog.com/talks/1-1-billion-evolution-document-serving-system/), as well.
